alibi_detect/saving/schemas.py

```diff
@@ -353,7 +349,11 @@ class KernelConfig(CustomBaseModelWithKwargs):
     "A string referencing a filepath to a serialized kernel in `.dill` format, or an object registry reference."
 
     # Below kwargs are only passed if kernel == @GaussianRBF
-    sigma: Optional[NDArray[np.float32]] = None
+    flavour: Literal['tensorflow', 'pytorch']
```

Comment:
Validation of this is essentially a 2-stage process. It is validated against the `Literal`, before being passed to `validate_framework` below to check against installed optional deps.